httpd/apache2/sites-enabled/registry.conf

```diff
@@ -12,6 +12,9 @@
     SSLCertificateFile /opt/ca/certs/apache.crt
     SSLCertificateKeyFile /opt/ca/private/apache.key
 
+    RequestHeader set X_FORWARDED_PROTO 'https'
+    SSLProxyEngine on
```

Comment:
Isn't this required if your backend is running https, i dont think we actually need it but i could be wrong.

Reply:
Registry greets with `ActionController::InvalidAuthenticityToken` if it's not provided